classes/template/PKPTemplateManager.inc.php

```diff
@@ -397,16 +382,6 @@ function initialize() {
 		));
 	}
 
-	/**
-	 * Override the Smarty {include ...} function to allow hooks to be
-	 * called.
-	 */
-	function _smarty_include($params) {
-		if (!HookRegistry::call('TemplateManager::include', array($this, &$params))) {
-			return parent::_smarty_include($params);
-		}
-		return false;
-	}
```

Comment:
I only found the FidusWriter plugin using this, but maybe just to give him a heads-up?

Reply:
Good idea. @johanneswilm, heads-up that the `TemplateManager::include` hook will be changing its conventions with the 3.2 release (late this year).

Reply:
thanks for letting me know!